src/components/Blogcontainer.jsx

```diff
@@ -13,62 +14,63 @@ function Blogcontainer() {
     setLike(isClick ? like - 1 : like + 1);
     setClick(!isClick);
   };
-  var addToList = (e) => {
+  let addToList = (e) => {
     e.preventDefault();
     setNewPostItem([...newPostItem, input.current.value]);
```

Comment:
instead of adding the post text as string, create a post object then push it to the array like the following:
```jsx
setPosts([...posts, {text: input.current.value, user: 'John Doe', likes: []}])
```